mirgecom/simutil.py

```diff
@@ -434,11 +434,20 @@ def create_parallel_grid(comm, generate_grid):
     return generate_and_distribute_mesh(comm=comm, generate_mesh=generate_grid)
 
 
+def configurate(config_key, config_object=None, default_value=None):
+    """Return a configured item from a configuration object."""
+    if config_object is not None:
+        if config_key in config_object:
+            return config_object[config_key]
```

Comment:
If it is like this, then it will continue to work for python dictionaries, but also for more general objects.

```suggestion
         if config_key in config_object.__dict__:
            return config_object.__dict__[config_key]
```

Reply:
I'm just wondering how that would work with a directly passed `dict`. `{}.__dict__` does not work.

Reply:
See my comment below https://github.com/illinois-ceesd/mirgecom/pull/716/files#r927776964